modules/sfp_ipregistry.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_ipregistry
+# Purpose:      IPRegistry database query module.
+#
+# Author:      Leo Trubach <leotrubach@gmail.com>
+#
+# Created:     2020-09-08
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import urllib.parse
+
+from spiderfoot import SpiderFootEvent, SpiderFootPlugin
+
+
+class sfp_ipregistry(SpiderFootPlugin):
+    __name__ = 'sfp_ipregistry'
+    meta = {
+        "name": "IP Registry",
```

Comment:
The name is technically `ipregistry` according to their website so I suggest we go with that. Please update this across the whole file.